aws/cloudformation/components/database.yml.erb

```diff
@@ -131,7 +131,8 @@
         # https://github.com/code-dot-org/code-dot-org/blob/654d0cf90dd65848fdd045df3b9e4f85903d9f68/pegasus/test/fixtures/fake_dashboard.rb#L257-L271
         # Disable binary logging to improve write performance.
         binlog_format: 'OFF'
-        gtid-mode: ON_PERMISSIVE
+        gtid_mode: 'ON'
```

Comment:
How long have we had this bug? 
<img width="675" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/2157034/9688214c-65a1-456f-81b2-3e16c4a1a564">

Reply:
OK. Not a defect. The ParameterGroup setting name is `gtid-mode`